drivers/usb/usbstor_new/usbstor.h

```diff
@@ -547,4 +547,6 @@ USBSTOR_TimerRoutine(
     PDEVICE_OBJECT DeviceObject,
      PVOID Context);
 
+#include <poppack.h>
```

Comment:
This poppack must be moved much more earlier above, to line 314, just after the `UFI_UNKNOWN_CMD` structure and before the hardware-unrelated `IRP_CONTEXT`.

Reply:
Why?

Reply:
**Why** would you want the non-hardware-related IRP_CONTEXT and other structures that follow it to be packed on 1-byte boundary, instead of the default platform one?

Reply:
Because that is what was done previously.